src/model_configuration.py

```diff
@@ -25,8 +25,8 @@ class ModelConfiguration:
     +--------------------+----------------------------------------------+---------------------------------------------+---------+
     | monte_carlo.N      | Number of Monte Carlo simulations            |                                             | 100     |
     +--------------------+----------------------------------------------+---------------------------------------------+---------+
-    | monte_carlo.on_what| List: Defines component to vary.             | 'Technologies', 'ImportPrices',             | Tec     |
-    |                    | Warning: Import/export prices                | 'ExportPrices'                              |         |
+    | monte_carlo.on_what| List: Defines component to vary.             | 'Technologies', 'Import',             | Tec     |
```

Comment:
We need to remove the model_configuration.py, right? it is not used anymore, I think

Reply:
Yes it was still here because Inge used it for documentation, and I changed this using CRTL SHIFT R so didn't notice the file. Do we want to delete the file in this branch?